dojo-dungeon/src/dojo/rooms/builder/RoomBuilder.java

```diff
@@ -127,7 +127,7 @@ public RoomBuilder sortables(HashMap<String, ArrayList<String>> sortables) {
    *
    * @return the newly generated room instance
    */
-  public Room buildRoom_Key() {
+  public Room buildRoom_FindKeyRoom() {
```

Comment:
@tgrothe wenn du schon die namen änderst, wieso passt du das dann nicht gleich richtig an? unterstriche sind in methodennamen in java nicht üblich.

Reply:
Ja, das stimmt. ... Aber ich verstehe zudem nicht (mehr) ganz, warum wir Build-Methoden in `RoomBuilder` _und_ in `DojoStarter` hatten (quasi parallel/doppelt). Kann ich das vielleicht auch mit ändern? Hättest du dazu eine Idee?

Reply:
> Ja, das stimmt. ... Aber ich verstehe zudem nicht (mehr) ganz, warum wir Build-Methoden in `RoomBuilder` _und_ in `DojoStarter` hatten (quasi parallel/doppelt). Kann ich das vielleicht auch mit ändern? Hättest du dazu eine Idee?

keine ahnung - ihr/du habt/hast das so eingebaut. vielleicht habe ich etwas falsch verstanden, aber bittest du mich gerade, dir deinen code zu erklären? 🫣

Reply:
nicht ganz 😅 ich bin mir nicht mehr ganz sicher, weshalb wir das so gemacht hatten.

aber ich könnte die builder-methoden von `RoomBuilder` nach `DojoStarter` verschieben - oder umgekehrt. dann hätte man diese nicht doppelt.

aber ich weiß nicht, ob das (a) sinvoll wäre, (b) in welche richtung und (c) ob ich das machen soll/darf. deshalb wäre eine kurze einschätzung von dir hilfreich (bevor ich zu viel "auf eigene faust" mache ...)

Reply:
jetzt sollte es besser lesbar und gemäß der koventionen sein.

Reply:
> aber ich weiß nicht, ob das (a) sinvoll wäre, (b) in welche richtung und (c) ob ich das machen soll/darf. deshalb wäre eine kurze einschätzung von dir hilfreich (bevor ich zu viel "auf eigene faust" mache ...)

naja, das hat mit diesem pr sicher nichts zu tun, wäre also ein eigener pr. 

und wenn du da eine deutliche verbesserung siehst, mit der du dich freiwillig in deiner freizeit beschäftigen möchtest, dann werde ich dir nicht im weg stehen.

Reply:
nein, alles gut. ich glaube, das "ist so ok" - wenngleich ich das gefühl habe, der starter tut etwas zu viel.

freizeit: guter hinweis, es ist woen. ;)

Reply:
@cagix Ich habe mir den Starter noch einmal angeschaut und eigentlich passt das so mit den Delegationsmethoden und Builder-Aufrufen. Also, aus meiner Sicht, kein Änderungsbedarf nötig. Nur ein erneutes Review wäre gut ...